api/DHSC.FingertipsNext.Api.IntegrationTests/DataManagement/DataManagementIntegrationTests.cs

```
@@ -134,23 +139,5 @@ public async Task UploadingNoFileShouldReturn400Response()
         response.StatusCode.ShouldBe(HttpStatusCode.BadRequest);
     }
 
-    [Fact]
-    public async Task UploadToBlobStorageShouldFailIfContainerDoesNotExist()
-    {
-        // Arrange
-        var apiClient = GetApiClient(_factory, "non-existent-container");
-
-        var blobContentFilePath = Path.Combine(TestDataDir, "blobContent.json");
-        await using var fileStream = File.OpenRead(blobContentFilePath);
-        using var content = new MultipartFormDataContent();
-        using var streamContent = new StreamContent(fileStream);
-        streamContent.Headers.ContentType = new MediaTypeHeaderValue("application/octet-stream");
-        content.Add(streamContent, "file", "blobContent.json");
-
-        // Act
-        var response = await apiClient.PostAsync(new Uri($"/indicators/{IndicatorId}/data", UriKind.Relative), content);
 
-        // Assert
-        response.StatusCode.ShouldBe(HttpStatusCode.InternalServerError);
-    }
```

Is this not still a valid test?

---

As the class should throw an error on application start-up, this is no longer a valid integration test.